src/training/params.py

```diff
@@ -326,13 +326,54 @@ def parse_args(args):
         action='store_true',
         help="Freeze BatchNorm running stats in image tower for any locked layers.",
     )
+    parser.add_argument(
+        "--lock-doc",
```

Comment:
what about a "lock-tower-1" and "lock-tower-2" param instead ?
query/doc is not the only thing

Reply:
Sure. I will rename these variables immediately. Do you think we should also rename variables like `image_features` in `train.py` and `main.py'? Will renaming variables cause confusion for the original text-image model? Currently `train.py` is fully compatible with TextTextCLIP. No modification other than renaming variables is needed.

Reply:
I have tested TextTextCLIP locally and it seems to work smoothly. But the script fails some tests here. I will make adjustments.